### ISSUE TITLE
(chromium) Outdated stable version

### ISSUE THREAD

**Issue by woolyss** [chocolatey/chocolatey-coreteampackages#1137]
Today, I am trying to install the latest STABLE version of Chromium but the installed version is outdated.

# Expected Behavior

Today the latest stable version of Chromium is 70.0.3538.77
Your updater works nicely but the install is wrong.

Your updater: https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/automatic/chromium/chromium.json --> OK
The installed version is: 64.0.3282.168 --> Outdated STABLE version.

![](https://i.imgur.com/yqm1YVe.png)

You can see the same outdated STABLE version on your site:
* https://chocolatey.org/packages?q=Chromium
* https://chocolatey.org/search?q=Chromium

![](https://i.imgur.com/14X5CsF.png)
![](https://i.imgur.com/9qvFb3S.png)

## Possible Solution
The latest stable version of Chromium should be 70.0.3538.77.

## Steps to Reproduce (for bugs)
To reproduce it, just install Chromium stable.
`choco install chromium-stable `

## Your Environment
* Package Version used: Chromium stable
* Operating System and version: Windows 10 Pro 1803
* Chocolatey version: v0.10.11 
* Install/uninstall gist:

**Comment by AdmiringWorm** [comment]
I think there may be some misunderstanding here.

The chromium-stable package is not the same as the chromium package maintained here in this repository.

The one you're actually looking for is just called [chromium](https://chocolatey.org/packages/chromium). When you don't use `--pre` when calling chocolatey, you'll get the latest stable version of chromium (with that argument, you'll get the latest snapshot).

**Comment by woolyss** [comment]
Hello AdmiringWorm,
No, I talk about the STABLE version, not the development version. Please, check my screenshots. They are all about the STABLE version.

**Comment by mikhaelkh** [comment]
@woolyss, afaik, [chroimum](https://chocolatey.org/packages/chromium) is about dev and stable versions, stable are default, dev with `--pre` switch.
[chromium-stable](https://chocolatey.org/packages/chromium-stable) hasn't updated since February 2018 so it's no point in keeping this outdated package on https://chromium.woolyss.com. [chroimum](https://chocolatey.org/packages/chromium) has 460K downloads, [chromium-stable](https://chocolatey.org/packages/chromium-stable) — 2.5K
What is not clear is why [chromium-stable](https://chocolatey.org/packages/chromium-stable) is still there with no warning of any kind when it's obvious it's outdated and abandoned, maybe @AdmiringWorm can explain this.

**Comment by AdmiringWorm** [comment]
@woolyss as @mikhaelkh said, the chromium package contains both stable and dev versions.

The `chromium-stable` package was created by a completely different maintainer, and isn't actually maintained in this repository at all (although the person maintaining that package, contributed to the current update script we're using here).

AFAIK, the chromium-stable package was actually supposed to be deprecated when the original chromium package managed to hit the actual stable release, which it have.

@mikhaelkh I can't actually answer that, since that package is maintained by someone else, that question needs to be directed to him.

Anyhow, below I'll try to answer some of your edited questions:

>  I agree with your suggestion but the chocolatey.org pages about Chromium are very unclear.

I do agree here, and I also know that this is caused by a bug in the website code.
What's supposed to happen, is that the latest stable version to be shown by default. I believe there is an open issue about this over at: https://github.com/chocolatey/chocolatey.org/issues
But unfortunately I'm unable to find it right now. If there isn't I would suggest to open an issue regarding this there.

> You should create better URL page like:
>
> chocolatey.org/packages/chromium about the STABLE version
> chocolatey.org/packages/chromium-snapshots about the DEV version

I don't create any urls, I only help out maintaining packages listed in this repository (as well as some in my own repository).
A new url, would actually result in a new package being created, or in other words a duplicate package (which is what we try to avoid when possible, which is why snapshot/dev builds is being provided with a pre-release version), and stable releases without it. (Unfortunately this wasn't always the case though)

> Or update your exisiting chocolatey.org/packages/chromium-stable page.

I'm sorry, but we can't update something we don't actually maintain.
This is something that needs to be directed to the maintainer of the chromium-stable package.

To give a little data with this, most of the packages on chocolatey.org is being provides by multiple maintainers (Most are volunteers, and not affiliated with chocolatey itself. Myself included). Everyone of them (well most of them anyhow) have their own repository where they maintain the packages they have pushed to chocolatey.org.


> There is another issue about the DEV version on the chocolatey.org site.
>
> Search chocolatey.org/packages?q=chromium
> The first answer will be Chromium 70.0.3538.77 (This is the STABLE version number!!!!???)
> You click on this link, it will open chocolatey.org/packages/chromium/70.0.3538.77 But it is a wrong result because this page should concern only DEV version. check Version History at the bottom page.
>
> Sorry, nothing is clear on this page too.
>
> I understand that you migrate different versions few month ago. But please, can you clarify that to help new users? Thank you very much.

Let me correct one thing here which I tried to explain in my previous comment.
The chromium package (the one you link to in the above quote) should contain both stable and development versions, all development versions have the following prefix added to the their version (`-snapshots`), which is done to prevent chocolatey to install development versions by default.

SO basically, yes that exact link you provided is meant to be the stable release.
And regarding the Version history, if you click on the `More` link and scroll down you'll actually see versions without the `snapshots` in the version.


> Moreover, nothing to do with this issue but you can update the Chromium logo too.
> commons.wikimedia.org/wiki/File:Chromium_Material_Icon.png

Thank you, I don't usually look at application icons, so I hadn't noticed this. I'll make sure it gets added to future package versions of chromium

**Comment by woolyss** [comment]
Thank you very much @AdmiringWorm and @mikhaelkh for your detailed explanation. All is clear now.  I will open an issue regarding the website issue about Chromium pages.

**Comment by AdmiringWorm** [comment]
> I will open an issue regarding the website issue about Chromium pages.

Please make sure that an issue doesn't exist already (I believe there may be one)

**Comment by AdmiringWorm** [comment]
Anyhow, I'll be closing this issue. As it isn't related to the package we're maintaining
